build-caching/src/main/java/com/vertispan/j2cl/build/PropertyTrackingConfig.java

```diff
@@ -141,6 +141,20 @@ public String getDependencyMode() {
         return Collections.emptySet();
     }
 
+    @Override
+    public Optional<File> getTranslationsFile() {
+        ConfigValueProvider.ConfigNode node = config.findNode("translationsFile");
+        if (node == null) {
+            return Optional.empty();
+        }
+
+        try {
+            return Optional.of(new File(node.readString()).getCanonicalFile());
```

Comment:
using readString means that only the string value of the path name is being tracked for changes - if the file itself is ever changed, only a clean rebuild will work (or deliberately change something unrelated)